storage/client/blob/src/main/java/com/azure/storage/blob/implementation/ServicesImpl.java

```diff
@@ -59,34 +61,34 @@ public ServicesImpl(AzureBlobStorageImpl client) {
      * proxy service to perform REST calls.
      */
     @Host("{url}")
-    @Service("Storage Blobs Service")
+    @ServiceInterface(name = "Services")
```

Comment:
Need a better name here - 'blobstorage' or something?

Reply:
I can do "Azure Blob Storage Services"

Reply:
I believe checkstyle expects it to have no spaces and be less than 20 characters. We can increase the character limit, but no spaces will be important. Use camel case if necessary.

Reply:
Using "AzureBlobStorageServices" right now